pages/events/index.vue

```diff
@@ -277,6 +320,82 @@ export default class extends QiskitPage {
     .bx--label {
       color: $cool-gray-80;
     }
+
+    @include mq($until: medium) {
+      display: none;
+    }
+
+  }
+
+  &__filters-region,
+  &__filters-type {
+    @include mq($from: medium) {
+      display: none;
+    }
+
+    .bx--list-box--light,
+    .bx--list-box__menu {
+      background-color: $white;
```

Comment:
We are restyling the multiselect here because it has not the [Carbon Design](http://vue.carbondesignsystem.com/?path=/story/components-cvmultiselect--default) colors of the light theme.
I don't know if there is something inherited, and it doesn't work properly.